### DISCUSSION TITLE
location Safetensors directory

### DISCUSSION THREAD

**Discussion by BraindamageBE** [nod-ai/SHARK#2147] [Q&A] [answered]
Hi, I recently switched to a new shark version due to hardware problems with my 5700XT after the juli 2023 release.
But now I can't seem to find the directory to put the safetensors files in.
It used to be just the "models" directory, but the files that I put there aren't recogniced by the app.
BUt also the loras I put in the models/loras folder don't seem to work, neither show up in the list in the app
is there a new directory for these files?

**Comment by monorimet** [comment, marked as answer]
models/checkpoints for custom SD/SDXL checkpoints; lora support is being reimplemented.

**Comment by BraindamageBE** [comment]
Hi, thanks for your reply.
Unfortunately I've put the files there, but the they don't show up.
The lora's where also in the correct folder
